nextcloud-startup-script.sh

```diff
@@ -39,16 +39,29 @@ then
     exit 1
 fi
 
+
+# Check network
+echo "Testing if network is OK..."
+sleep 2
+sudo ifdown $IFACE && sudo ifup $IFACE
```

Comment:
Usually when i use VPS i connect to it via SSH and i think that turn off the network interface where i'm connected It is not a great idea 😄